source/jormungandr/tests/check_utils.py

```diff
@@ -1348,3 +1348,7 @@ def make_crit(func, reverse=False):
         make_crit(lambda j: get_valid_int(j['duration'])),
         make_crit(lambda j: len(j.get('sections', []))),
     ]
+
+
+def has_the_disruption(response, disrupt_id):
+    return len([d['id'] for d in response['disruptions'] if d['id'] == disrupt_id]) != 0
```

Comment:
Using `any` would simplify the code/reading.

```suggestion
    return any([d['id'] for d in response['disruptions'] if d['id'] == disrupt_id])
```